generator.yaml

```diff
@@ -44,11 +44,16 @@ resources:
       QueueArn:
         is_attribute: true
         is_arn: true
+        is_read_only: true
         type: string
       QueueName:
         is_required: true
+        is_immutable: true
         type: string
       Tags:
         compare:
           is_ignored: true
+      QueueUrl:
+        is_readonly: true
```

Comment:
This should be `is_read_only` :)

This actually isn't necessary, since the `QueueUrl` field is returned in the output shape of the CreateQueue API and therefore will be placed in the Status (which is equivalent to `is_read_only`=true)